copybins.sh

```diff
@@ -81,3 +84,13 @@ if [[ $pkr_version && $pkr_image ]]; then
     docker cp packer:/usr/local/bin/ pkr_files
   fi
 fi
+
+# Include the promote artifact tool into container
+if [[ -n "$circle_token" ]]; then
+    curl https://circleci.com/api/v1.1/project/github/${artifact_tool}/${artifact_tool_version}/artifacts?circle-token=$circle_token | grep -o 'https://[^"]*' | \
+    while read bin; do
+        bin_name=`basename $bin`
+        curl ${bin}?circle-token=$circle_token > /usr/local/bin/${bin_name}
```

Comment:
Are you trying to copy the binaries from one container to the ci container? If so you need to do a docker cp after running the container in question first to make sure its there like what is being done with terraform and packer containers. If your trying to put a binary just in /usr/local/bin in the container then you would need to copy the binary in the Dockerfile during build with the entrypoint.sh script after making it available to circleci in your home directory.